lib/rubocop/cop/rails/top_level_include.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks that `include`, `extend` and `prepend` exists at
+      # the top level.
+      # Using these at the top level affects the behavior of `Object`.
+      # Under the `app` directory of the Rails application, the convention
+      # that a file name and a class name are mapped is applied. There will
+      # not be using `include`, `extend` and `prepend` at the top level.
+      # Let's use it inside `class` or `module`.
+      #
+      # @example
+      #   # bad
+      #   include M
+      #
+      #   class C
+      #   end
+      #
+      #   # bad
+      #   extend M
+      #
+      #   class C
+      #   end
+      #
+      #   # bad
+      #   prepend M
+      #
+      #   class C
+      #   end
+      #
+      #   # good
+      #   class C
+      #     include M
+      #   end
+      #
+      #   # good
+      #   class C
+      #     extend M
+      #   end
+      #
+      #   # good
+      #   class C
+      #     prepend M
+      #   end
+      class TopLevelInclude < Cop
+        MSG = '`%s` is used at the top level. Use inside `class` or `module`.'
```

Comment:
Please use the annotated form for the format string, e.g.:

```ruby
`%<keyword>s`
```

(I think someone was working on a cop for this. 🤔)

Reply:
Yeah, there's such a cop indeed. Not sure what we set it do, though.